git/split-up-a-commit.md

```diff
@@ -0,0 +1,47 @@
+# Split Up a Commit, Rewrite History
+
+When working on a branch with multiple commits, you can "go back in time" and revise previous commits any way you please.
+
+    $ git rebase -i origin/master
+
+This command will prompt you inside of your `$EDITOR` with a series of commit SHAs and commit titles
```

Comment:
Is it in `$EDITOR` or `$VISUAL`? http://robots.thoughtbot.com/visual-ize-the-future